tests/ansible_lint_rules/NoFormattingInWhenRule.py

```diff
@@ -1,20 +1,19 @@
-# Documentation on AnsibleLintRules can be found here:
```

Comment:
I'm trying to keep this file in sync with the PR upstream. Will had requested some changes, mainly around Python 3 compatibility, so I pulled them in here when I added the until code, so I could push the until code back into that PR.

Basically it's just a bunch of back and forth until it reaches its final form.